tests/unit/ext/switches/brocade.py

```diff
@@ -219,9 +221,9 @@ def test_apply_networking(self, switch, nic, network):
             switch.apply_networking(action_native)
 
             assert mock.called
-            assert mock.call_count == 3
-            assert mock.request_history[0].text == TRUNK_PAYLOAD
-            assert mock.request_history[2].text == TRUNK_NATIVE_PAYLOAD
+            assert mock.call_count == 4
+            assert mock.request_history[1].text == TRUNK_PAYLOAD
```

Comment:
Should we still be checking `request_history[0]`? I'm fuzzy on exactly how this test works, but I would expect adding a new request we should check it like the others.